src/features/governance/components/CurrentVotingBalanceDetails.tsx

```diff
@@ -43,10 +45,14 @@ const Detail = ({
       </div>
       {Boolean(unlock) && (
         <div className="flex gap-2 items-center justify-between mb-2">
-          <i className="inline-flex">of which {value?.convertToBase().toFixed()} can be unlocked</i>
-          <Button onClick={unlock} classes="h-[24px]">
-            <Pill classes={inModal ? 'bg-primary' : 'bg-secondary'}>Unlock</Pill>
-          </Button>
+          <i className="inline-flex">
+            of which {unlockable?.convertToBase().toFixed()} can be unlocked
```

Comment:
so this can be `0`, right?

Reply:
correct, just wanted to be explicit about it